iacrcc/tests/compile_tests.py

```diff
@@ -258,18 +258,18 @@ def test11_test():
       assert meta['funders'][2]['fundref'] == '517622'
       assert meta['funders'][2]['grantid'] == '57821-3'
       assert 'country' not in meta['funders'][2]
-      pdfpath = tmpdirpath + '/main.pdf'
-      xmp = XMPParser(pdfpath)
-      description = xmp.get_string('.//dc:description/rdf:Alt/rdf:li')
-      assert description == 'IACR Communications in Cryptology, DOI:XXXXXXXX'
-      assert xmp.get_string('.//dc:identifier') == 'info:doi/XXXXXXXX'
-      assert xmp.get_string('.//prism:publicationName') == 'IACR Communications in Cryptology'
-      assert xmp.get_string('.//prism:doi') == 'XXXXXXXX'
-      assert xmp.get_string('.//prism:publicationName') == 'IACR Communications in Cryptology'
-      assert xmp.get_string('.//prism:aggregationType') == 'journal'
-      assert xmp.get_string('.//dc:source') == 'main.tex'
-      authors = xmp.get_strings('.//dc:creator/rdf:Seq/rdf:li')
-      assert len(authors) == 2
+      #pdfpath = tmpdirpath + '/main.pdf'
```

Comment:
We should be able to at least verify that the XMP contains `dc:subject`, `dc:creator` as a comma delimited string, `dc:subject` with keywords, and `dc:rights` with the license. I propose that we defer the update of the tests until a subsequent PR.